.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
```

Comment:
The commit message could just be `coafile: Move PyUnusedCodeBear to its own section` or something like that.